app/vmui/packages/vmui/src/components/DownloadButton/DownloadButton.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC, useCallback, useState } from "preact/compat";
+import Tooltip from "../Main/Tooltip/Tooltip";
+import Button from "../Main/Button/Button";
+import { DownloadIcon } from "../Main/Icons";
+import Popper from "../Main/Popper/Popper";
+import { useRef } from "react";
+import "./style.scss";
+
+interface DownloadButtonProps {
+    title: string;
+    downloadFormatOptions?: string[];
+    onDownload: (format?: string) => void;
+}
+
+const DownloadButton: FC<DownloadButtonProps> = ({ title, downloadFormatOptions, onDownload }) => {
+  const [isPopupOpen, setIsPopupOpen] = useState<boolean>(false);
+  const onClosePopup = useCallback(() => setIsPopupOpen(false), []);
```

Comment:
Can use `useBoolean` (from `src/hooks/useBoolean.ts`) for cleaner state handling:  
```ts
const {
  value: isPopupOpen,
  setTrue: onOpenPopup,
  setFalse: onClosePopup,
} = useBoolean(false);

Reply:
fix it